Lagerregal/base_settings.py

```diff
@@ -135,9 +135,9 @@
 
 SITE_NAME = "Lagerregal"
 
-MESSAGE_TAGS = {
-    messages.ERROR: 'alert',
-}
+# MESSAGE_TAGS = {
+#     messages.ERROR: 'alert',
+# }
```

Comment:
what's happening here?

Reply:
That stopped the error messages from having the right color, looks like it was inserted to customize the error, but I couldn't find any reason for it, so I commented it out instead of deleting it... Should I delete it?